### PR TITLE
Update webhook api && fix readme

### PR DESCRIPTION
**Pull request by lavrovas**
- Fix readme typos
- Update webhook API for someone not willing to receive usernames and photos

### CODE REVIEW

File: aioviberbot/api/api.py [hunk 1]
```diff
@@ -30,9 +30,9 @@ def name(self):
     def avatar(self):
         return self._bot_configuration.avatar
 
-    async def set_webhook(self, url, webhook_events=None, is_inline=False):
+    async def set_webhook(self, url, webhook_events=None, send_name=True, send_photo=True, is_inline=False):
```
Comment (antonmyronyuk) [inline aioviberbot/api/api.py]: Hi, thank you for the PR!

Let's change order of function params here: `send_name` and `send_photo` should be after exisiting `is_inline` param to keep backward compatibility (if someone uses non-keywords args for this method)

File: aioviberbot/api/api_request_sender.py [hunk 2]
```diff
@@ -76,10 +76,12 @@ async def post_request(self, endpoint, payload=None):
             if not self._client_session:
                 await session.close()
 
-    async def set_webhook(self, url, webhook_events=None, is_inline=False):
+    async def set_webhook(self, url, webhook_events=None, send_name=True, send_photo=True, is_inline=False):
```
Comment (antonmyronyuk) [inline aioviberbot/api/api_request_sender.py]: Params order should be changed here as well

**Review by antonmyronyuk — CHANGES_REQUESTED** [review]

File: aioviberbot/api/api.py (hunk 1 above)
Comment (lavrovas) [reply aioviberbot/api/api.py]: By the way I cannot find anything about `is_inline` param in the API description, looks like it was removed a while ago.

Comment (antonmyronyuk) [reply aioviberbot/api/api.py]: Got it, I'll remove it in next releases

**Comment by antonmyronyuk** [comment]
Released in version `0.6.0` https://pypi.org/project/aioviberbot/0.6.0/

**Pull request merged**